Apps/Contoso.WinUI.Desktop.Demo/Contoso.WinUI.Desktop.Demo/App.xaml.cs

```diff
@@ -0,0 +1,153 @@
+using Windows.ApplicationModel.Core;
+using Windows.Storage;
+using System.Globalization;
+using Microsoft.AspNetCore.StaticFiles;
+
+// To learn more about WinUI, the WinUI project structure,
+// and more about our project templates, see: http://aka.ms/winui-project-info.
+
+namespace Contoso.WinUI.Desktop.Demo
+{
+    /// <summary>
+    /// Provides application-specific behavior to supplement the default Application class.
+    /// </summary>
+    public partial class App : Application
+    {
+        private ApplicationDataContainer localSettings;
+
+        /// <summary>
+        /// Initializes the singleton application object.  This is the first line of authored code
+        /// executed, and as such is the logical equivalent of main() or WinMain().
+        /// </summary>
+        public App()
+        {
+
+            // Init settings.
+            localSettings = ApplicationData.Current.LocalSettings;
+            TaskScheduler.UnobservedTaskException += (object sender, UnobservedTaskExceptionEventArgs args) =>
+            {
+                // If you see this message while testing the app and if the stack trace is SDK related, we might have a bug in the SDK as we don't want to leak any exception from the SDK.
+                AppCenterLog.Error("AppCenterPuppet", "Unobserved exception observed=" + args.Observed, args.Exception);
+            };
+            CoreApplication.EnablePrelaunch(true);
+            InitializeComponent();
+            AppCenter.LogLevel = LogLevel.Verbose;
+            AppCenter.SetLogUrl("https://in-integration.dev.avalanch.es");
+
+            // Set data from local storage.
+            var countryCode = localSettings.Values[Constants.KeyCountryCode] as string;
+            if (!string.IsNullOrEmpty(countryCode))
+            {
+                AppCenter.SetCountryCode(countryCode);
+            }
+            var storageSize = localSettings.Values[Constants.KeyStorageMaxSize] as long?;
+            if (storageSize != null && storageSize > 0)
+            {
+                AppCenter.SetMaxStorageSizeAsync((long)storageSize);
+            }
+
+            // User callbacks.
+            Crashes.ShouldProcessErrorReport = (report) =>
+            {
+                Log($"Determining whether to process error report with an ID: {report.Id}");
+                return true;
+            };
+            Crashes.GetErrorAttachments = GetErrorAttachmentsHandler;
+
+            // Event handlers.
+            Crashes.SendingErrorReport += (_, args) => Log($"Sending error report for an error ID: {args.Report.Id}");
+            Crashes.SentErrorReport += (_, args) => Log($"Sent error report for an error ID: {args.Report.Id}");
+            Crashes.FailedToSendErrorReport += (_, args) => Log($"Failed to send error report for an error ID: {args.Report.Id}");
+
+            // Start App Center.
+            AppCenter.Start("40814a80-0782-4d5e-b9de-698935156d55", typeof(Analytics), typeof(Crashes));
```

Comment:
Let's use a constant.

Reply:
Not sure that it's important because it is used only in one place
JFYI: We have similar implementation in all other test apps.